src/github.sh

```diff
@@ -19,12 +20,18 @@ github::calculate_total_modifications() {
       ((deletions += $(echo "$body" | jq '.deletions')))
     fi
   else
+    # NOTE: this code is not resilient to changes w/ > 100 files as we're not paginating
     local -r body=$(curl -sSL -H "Authorization: token $GITHUB_TOKEN" -H "$GITHUB_API_HEADER" "$GITHUB_API_URL/repos/$GITHUB_REPOSITORY/pulls/$pr_number/files?per_page=100")
 
     for file in $(echo "$body" | jq -r '.[] | @base64'); do
       filename=$(jq::base64 '.filename')
+      status=$(jq::base64 '.status')
       ignore=false
 
+      if [ "$ignore_file_deletions" == "true" ] && [ "$status" == "removed" ]; then
```

Comment:
A tiny perf improvement. If all lines are ignored then no point in continuing the loop if the entire file is deleted.
```suggestion
      if [[ ( "$ignore_file_deletions" == "true" || "$ignore_line_deletions" == "true" ) && "$status" == "removed" ]]; then
```

Reply:
yeah good call, slight improvement